Makefile

```
@@ -1,3 +1,5 @@
+# Clang 15 and below have incomplete C++20 support.
+CXX = clang++-16
```

```suggestion
CXX = clang++
```
I'd rather not hardcode a version postfix, but I'll add in a readme soonish so the version limitation isn't restricted to this one note though